src/prefect/server/models/deployments.py

```diff
@@ -570,51 +570,59 @@ async def _generate_scheduled_flow_runs(
     """
     runs = []
 
-    # retrieve the deployment
     deployment = await session.get(db.Deployment, deployment_id)
 
-    if not deployment or not deployment.schedule or not deployment.is_schedule_active:
+    if not deployment:
```

Comment:
would you mind to explain why if there's no `deployment.schedule` we would still want to attempt to read the deployment schedules?

Reply:
This context may be buried, but we are migrating everyone from having one schedule in deployment.schedule to having one schedule in the now-multi-value deployment_schedules table.